src/jit/compiler.h

```diff
@@ -1133,7 +1133,8 @@ struct FuncInfoDsc
                                // funclet. It is only valid if funKind field indicates this is a
                                // EH-related funclet: FUNC_HANDLER or FUNC_FILTER
 
-#if defined(_TARGET_AMD64_)
+#if FEATURE_EH_FUNCLETS
+#if defined(_TARGET_XARCH_)
```

Comment:
As x86/Linux uses ``RUNTIME_FUNCTION`` with ``EndAddress``, we need to update its value inside ``CEEJitInfo::allocUnwindInfo``.

For that purpose, we need to store and deliver various ``emitLocation`` variables from JIT to EE.